onnxruntime/test/mlas/unittest/test_transpose.cpp

```diff
@@ -46,13 +46,15 @@ class MlasTransposeTest : public MlasTestBase {
 };
 
 template <> MlasTransposeTest<uint32_t>* MlasTestFixture<MlasTransposeTest<uint32_t>>::mlas_tester(nullptr);
+template <> MlasTransposeTest<uint16_t>* MlasTestFixture<MlasTransposeTest<uint16_t>>::mlas_tester(nullptr);
```

Comment:
The description mentioned.
> Matrix transpose for 16b tensors (shorts, and half precision floats)
Could you add tests for float16?

Reply:
It's exact the same with uint16_t. You can check the instructions used, it does not involve any computation, just data organization. So as long as the data element size is the same, the result is the same.